### PR TITLE
Adst 547

### PR DESCRIPTION
**Pull request by PiplaniVinay**
Implemented Auth0 using the OIDC and Removed the extra code.

### CODE REVIEW

File: AlfrescoAuth/Sources/Model/AlfrescoCredential.swift [hunk 1]
```diff
@@ -48,7 +48,11 @@ public struct AlfrescoCredential: Codable {
             case "expires_in":
                 accessTokenExpiresIn = value as? Int
             case "refresh_token":
-                refreshToken = value as? String
+                if let intValue = value as? Int {
```
Comment (gl-lovekesh) [inline AlfrescoAuth/Sources/Model/AlfrescoCredential.swift:51]: Create a method and return the desired output.

**Review by aman-alfresco — APPROVED** [review]

File: AlfrescoAuth/Sources/Model/AlfrescoCredential.swift (hunk 1 above)
Comment (PiplaniVinay) [reply AlfrescoAuth/Sources/Model/AlfrescoCredential.swift]: Done, Please Review

Comment (PiplaniVinay) [reply AlfrescoAuth/Sources/Model/AlfrescoCredential.swift]: > Create a method and return the desired output.

Done, Please review

**Review by gl-lovekesh — CHANGES_REQUESTED** [review]

File: AlfrescoAuth/Sources/Model/AlfrescoCredential.swift [hunk 2]
```diff
@@ -71,7 +75,13 @@ public struct AlfrescoCredential: Codable {
         tokenType = try values.decode(String.self, forKey: .tokenType)
         accessToken = try values.decode(String.self, forKey: .accessToken)
         accessTokenExpiresIn = try values.decode(Int.self, forKey: .accessTokenExpiresIn)
-        refreshToken = try values.decode(String.self, forKey: .refreshToken)
+        if let intValue = try? values.decode(Int.self, forKey: .refreshToken) {
```
[the remark this answers is not in the document]
Comment (PiplaniVinay) [reply AlfrescoAuth/Sources/Model/AlfrescoCredential.swift:87]: Done, Please Review

**Review by gl-lovekesh — APPROVED** [review]

**Pull request merged**